gcloud/bigquery/job.py

```diff
@@ -0,0 +1,702 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Define API Jobs."""
+
+import six
+
+from gcloud.exceptions import NotFound
+from gcloud._helpers import _datetime_from_microseconds
+from gcloud.bigquery.table import SchemaField
+from gcloud.bigquery.table import _build_schema_resource
+from gcloud.bigquery.table import _parse_schema_resource
+
+
+class _LoadConfiguration(object):
+    """User-settable configuration options for load jobs."""
+    # None -> use server default.
+    _allow_jagged_rows = None
+    _allow_quoted_newlines = None
+    _create_disposition = None
```

Comment:
I envision that this class will be reused by the other job classes that implement other types of table loading (e.g., via `/upload` w/ multipart or resumable options).